test/mocks/data/ranking_data.dart

```diff
@@ -0,0 +1,46 @@
+import "package:collection/collection.dart";
+import "package:proxima/models/ui/ranking/ranking_details.dart";
+import "package:proxima/models/ui/ranking/ranking_element_details.dart";
+
+import "firestore_user.dart";
+
+/// Mock user list for the ranking widget.
+final List<Map<String, int>> _rankingUserList = [
```

Comment:
Good idea to duplicate data here (instead of reusing from UI) so that when integrated with viewmodel this test will still work !